redos/7/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7 as prepare
```

Comment:
How about ABI compatibility between RedOS 7.2 and RedOS 7.3? Shouldn't we place the docker file into `redos/7.3` instead of `redos/7` and tag the image as `packpack/packpack:redos-7.3`?

I don't know, just asking.

Reply:
It seems, it is resolved. Now the directory is named as `redos/7.3`.